fixtures/import_data.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+from openfoodfacts import Flavor
+from sqlalchemy.orm import Session
+
+from app.db import Base, engine
```

Comment:
when i tried to run it locally - `poetry run python fixtures/import_data.py` - i get the following error
```
Traceback (most recent call last):
  File "....../open-prices/fixtures/import_data.py", line 5, in <module>
    from app.db import Base, engine
ModuleNotFoundError: No module named 'app'
```